shaka/src/js/dom/exception_code.h

```diff
@@ -20,31 +20,39 @@
 namespace shaka {
 
 enum ExceptionCode {
```

Comment:
There is already an accurate link referenced in `dom_exception.cc`, should I copy it here? It contains all of the same error codes in the same order with the exception of `NotAllowedError` being omitted. Previously https://developer.mozilla.org/en-US/docs/Web/API/DOMException was referenced, which is where this entry originated. I also added `InUseAttributeError` since it was missing and not referenced on the MDN page.